dind-golang/test.yaml

```diff
@@ -0,0 +1,11 @@
+---
+docker:
+  entrypoint: top
+tests:
+  root_user_test: False
```

Comment:
@danielhoherd  this flag is controlling the root user test for particular docker image.